x/gov/legacy/v040/migrate.go

```diff
@@ -77,23 +85,61 @@ func migrateContent(oldContent v036gov.Content) *codectypes.Any {
 		}
 	case v036distr.CommunityPoolSpendProposal:
 		{
-			protoProposal := &v040distr.CommunityPoolSpendProposal{
+			protoProposal = &v040distr.CommunityPoolSpendProposal{
 				Title:       oldContent.Title,
 				Description: oldContent.Description,
 				Recipient:   oldContent.Recipient.String(),
 				Amount:      oldContent.Amount,
 			}
-			// Convert the content into Any.
-			contentAny, err := codectypes.NewAnyWithValue(protoProposal)
-			if err != nil {
-				panic(err)
+		}
+	case v038upgrade.CancelSoftwareUpgradeProposal:
+		{
+			protoProposal = &v040upgrade.CancelSoftwareUpgradeProposal{
+				Description: oldContent.Description,
+				Title:       oldContent.Title,
+			}
+		}
+	case v038upgrade.SoftwareUpgradeProposal:
+		{
+			protoProposal = &v040upgrade.SoftwareUpgradeProposal{
+				Description: oldContent.Description,
+				Title:       oldContent.Title,
+				Plan: v040upgrade.Plan{
+					Name:   oldContent.Plan.Name,
+					Time:   oldContent.Plan.Time,
+					Height: oldContent.Plan.Height,
+					Info:   oldContent.Plan.Info,
+				},
+			}
+		}
+	case v036params.ParameterChangeProposal:
```

Comment:
v036params's ParamChange struct has 4 field:
https://github.com/cosmos/cosmos-sdk/blob/118aa614c5d5e81e3622336a61d053fec26c58c8/x/params/types/proposal.go#L81-L86

Our new proto type only has 3 fields:
https://github.com/cosmos/cosmos-sdk/blob/fdcb028636c940b94387ad86b83959aab4b92cde/proto/cosmos/params/v1beta1/params.proto#L21-L27

Just wanted to confirm: is this on purpose? Or did we forget a field in the proto migration?

Reply:
Looks like we forgot to add it. @alexanderbez ?

Reply:
I'm pretty sure it was removed on purpose